core/templates/pages/exploration-editor-page/feedback-tab/thread-table/thread-table.component.ts

```diff
@@ -12,23 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Component, Input } from '@angular/core';
-import { downgradeComponent } from '@angular/upgrade/static';
-import { SuggestionThread } from 'domain/suggestion/SuggestionThreadObjectFactory';
-import { DateTimeFormatService } from 'services/date-time-format.service';
-import { ThreadStatusDisplayService } from '../services/thread-status-display.service';
-
 /**
  * @fileoverview Directive for displaying the list of threads in the feedback
  * tab of the exploration editor.
  */
 
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { SuggestionThread } from 'domain/suggestion/SuggestionThreadObjectFactory';
+import { DateTimeFormatService } from 'services/date-time-format.service';
+import { ThreadStatusDisplayService } from '../services/thread-status-display.service';
+
 @Component({
-  selector: 'thread-table',
+  selector: 'oppia-thread-table',
```

Comment:
When I asked for a selector change, this would require changes in line number 54 as well. That would become `oppiaThreadComponent`. Then in all the HTML file, change the tag from `<thread-table ` to `<oppia-thread-table`

Reply:
Done.